tw-tkms-test-starter/build.gradle

```diff
@@ -15,13 +15,13 @@ dependencies {
     annotationProcessor libraries.spotbugsAnnotations
 
     compileOnly libraries.springBootConfigurationProcessor
-
+    
     implementation project(":tw-tkms-starter")
 
     implementation libraries.commonsLang3
     implementation libraries.jacksonDatabind
     implementation libraries.jakartaValidationApi
```

Comment:
I guess the jakartaValidation can be removed here as well.

Reply:
Removed.